src/math/Matrix3.js

```diff
@@ -152,25 +153,18 @@ THREE.Matrix3.prototype = {
 
 	getInverse: function ( matrix, throwOnDegenerate ) {
 
-		// input: THREE.Matrix4
-		// ( based on http://code.google.com/p/webgl-mjs/ )
+		var me = matrix.elements,
+			te = this.elements,
 
-		var me = matrix.elements;
-		var te = this.elements;
+			n11 = me[ 0 ], n21 = me[ 1 ], n31 = me[ 2 ],
+			n12 = me[ 3 ], n22 = me[ 4 ], n32 = me[ 5 ],
```

Comment:
Local variables are ethereal and not subjected to garbage collection. They may hold new object (they don't in this case) in which case *the objects* get collected once they are no longer referenced anywhere.